caddyhttp/httpserver/plugin.go

```diff
@@ -490,6 +490,8 @@ var directives = []string{
 	"hugo",      // github.com/hacdias/caddy-hugo
 	"mailout",   // github.com/SchumacherFM/mailout
 	"awslambda", // github.com/coopernurse/caddy-awslambda
+
```

Comment:
Where ever it works correctly and doesn't break other middlewares in the chain. We can always move it later, but I always ask to double check and make sure that the author at least considered that the plugin is in the right place. Whatever makes the most logical sense in the order of things done by handlers, really.